src/components/PageTitles.js

```diff
@@ -41,16 +48,44 @@ const PageTitles = {
   defaultPageTitle: "Register a Food Business"
 };
 
-PageTitles.getUrlPageTitle = (url) => {
+PageTitles.getUrlPageTitle = (
+  url,
+  language,
+  validatorErrors,
+  allValidationErrors
+) => {
+  var allValidationErrorLength =
+    typeof allValidationErrors === "object"
+      ? Object.keys(allValidationErrors).length
+      : 0;
+  var validatorErrorLength =
+    typeof validatorErrors === "object"
+      ? Object.keys(validatorErrors).length
+      : 0;
+
+  var errorLength =
+    allValidationErrorLength > validatorErrorLength
+      ? allValidationErrorLength
+      : validatorErrorLength;
   var urlParts = url.split("/");
   var page = (urlParts[2] ?? urlParts[1]).split("?")[0];
   var title = PageTitles.defaultPageTitle;
+  var translatedTitles =
+    language === "cy"
+      ? errorLength > 0
+        ? cyErrorTitles
+        : cyTitles
+      : errorLength
+      ? enErrorTitles
+      : enTitles;
 
   if (page && page in PageTitles.pageTitles) {
-    title = `${PageTitles.prefix} - ${PageTitles.pageTitles[page]}`;
+    title =
+      errorLength > 0
+        ? `Error: ${PageTitles.prefix} - ${PageTitles.pageTitles[page]}`
+        : `${PageTitles.prefix} - ${PageTitles.pageTitles[page]}`;
   }
-
-  return title;
+  return translatedTitles[title] || title;
```

Comment:
None of this translation should need to be done here.  Just return the english and the translation is done in the _app.js